### ISSUE TITLE
Invitation message not translated

### ISSUE THREAD

**Issue by manu-p** [jitsi/jitsi-meet#4884]
## Description
Invitation message still in english when UI's not in english.

## Current behavior
When a non english speaking user, let's say french, with France set in his parameters UI, wants to share the meeting link via the "information" button, the message copied is in english, not french.

## Expected Behavior
Invitation message in local language, not english.

## Possible Solution
Paste the invitation message and replace "is invitiing you to a meeting." and "Join the meeting:" by their equivalent in french.

# Environment details
main-fr.json exists and the "info" section seems not to be used...

**Comment by damencho** [comment]
Please, when having questions and problems first discuss them in the community forum, before opening a new issue.

I just tested this on beta and on meet.jit.si and it works as expected in french:
```
Vous êtes invité(e) à participer à une réunion.
Rejoindre la réunion:
https://meet.jit.si/.....
```
Where do you experience this?

**Comment by saghul** [comment]
Thanks for the report! You can contribute the translation easily, check this: https://github.com/jitsi/jitsi-meet/tree/master/lang#translating-jitsi-meet once merged it will be part of the next release.

**Comment by manu-p** [comment]
Thanks @damencho sorry for the inconvenience and thanks for replying.

The jitsi instance is https://meet.combustible-numerique.fr, and it seems to be a bit more complicated than what I thought... 

Yes, it works, you're right. And I just tested right now creating a new meeting and establishing the invitation message is well in french. 

Before then, I've been using this same instance through a NextCloud embedded external link, that is integrated in NC as if it was an application, and the invitation message remained in english, while the whole setup (FF, NC, Jitsi) was in french. 
Now that I've been using the instance "outside" NC and checked the invitation message was indeed in french, back then to the NC integrated use, the message is now in french... 

In other words, forget it! :-D
